ts/components/LabelledItem/index.tsx

```diff
@@ -85,7 +87,7 @@ type LabelColor = Exclude<DescriptionColor, "red">;
 type ColorByProps = {
   borderColor: string | undefined;
   descriptionColor: DescriptionColor;
-  iconColor: string;
+  iconColor: IOColors;
   labelColor: LabelColor;
   placeholderTextColor: string;
 };
```

Comment:
I believe this file could use a little refactoring by removing the `React.FC` and `function` syntaxes in favour of type inferring and arrow syntax respectively. 
also, I think the type definitions in line 101-105 should be extracted to their own type; what do you think?
Overall, I believe this file is mostly legacy so it would benefit from a little cleanup, maybe it would be a good idea to add a task and tag it as technical debt.

Reply:
@forrest57 This component is obsolete and will soon be replaced by the new one (https://github.com/pagopa/io-app-design-system/pull/44), so I wouldn't spend much time on a component that we know has a short lifespan.